build.gradle

```diff
@@ -32,11 +32,13 @@ buildscript {
         classpath 'org.springframework.boot:spring-boot-gradle-plugin:1.5.15.RELEASE'
         classpath 'io.franzbecker:gradle-lombok:1.14'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath "com.github.ben-manes:gradle-versions-plugin:0.20.0"
```

Comment:
nit. Please use single quote for consistency.

Reply:
Thank you. Updated